app/routes/influencer.js

```diff
@@ -4,13 +4,17 @@ require('../middleware/authentication.js');
 
 var influencerController = require('../middleware/influencer');
 var passport = require('passport');
+var Signup = require('../models/signup');
 
 module.exports = function(router) {
     router.post('/signup', influencerController.addInfluencer);
 
     router.get('/signup/:signupId', (request, response) => {
         if (!request.user) {
-            response.render('signup', {logged_in: request.user != null});
+            Signup.findOne({signupLink: request.params.signupId})
```

Comment:
I added this because I'm passing the signup object to the context for the template. The original idea was for two reasons: 1. I wanted to pass in the signupLink key to the form so we can use it in the form submission rather than depending on splitting the URL (see the changes to signup.hbs and core.js). 2. I was trying to add the email address into the form as a read-only field (since we're not allowing the user to signup with any email they want, but only the one tied to the signup document). I actually removed this temporarily because Angular wasn't submitting a disabled form field, so I need to find a different way to do that in a future PR.

This should still allow you to get to the signup page, just that you'll pass in `undefined` to the template context. Was there something else you were trying to do in testing that won't work with this?